run.py

```diff
@@ -47,21 +47,48 @@ async def on_ready():
         await General.udisconnect(self=None, ctx=None, guild=guild)
 
         try:
-            await guild_to_audiocontroller[guild].register_voice_channel(guild.voice_channels[0])
+            await current_guild.voice_client.disconnect(force=True)
         except:
             pass
 
         print("Joined {}".format(guild.name))
+        print('We have logged in as {0.user}'.format(bot))
 
     print(config.STARTUP_COMPLETE_MESSAGE)
 
 
+#embed_obj = discord.Embed(description = "test")
+
+
 @bot.event
 async def on_guild_join(guild):
-    print(guild.name)
+    print("Join to " + guild.name)
     guild_to_audiocontroller[guild] = AudioController(bot, guild)
-
     await guild_to_audiocontroller[guild].register_voice_channel(guild.voice_channels[0])
 
+@bot.event
+async def on_guild_join(guild):
+    await guild.text_channels[0].send(config.JOIN_MASSAGE)
+
+#    await bot.change_presence(status=discord.Status.idle, activity=discord.Game(name="{}info".format(config.BOT_PREFIX)))
+
+#    config.BOT_VERISON = "0.2.35"
+
+    for guild in bot.guilds:
+
+        guild_to_audiocontroller[guild] = AudioController(bot, guild)
+
+        await guild_to_audiocontroller[guild].register_voice_channel(guild.voice_channels[0])
+
+        await General.udisconnect(self=None, ctx=None, guild=guild)
+
+        try:
+            await current_guild.voice_client.disconnect(force=True)
+        except:
+            pass
+        print('New Server')
+
+
+
```

Comment:
Targeted issue discovered to be existing issue with youtube-dl based on submitted logs.

**Some recommendations:**
- Bot should not reset all connections on guild join. This is bad practice and will reset playlists/audio for all guilds the bot is in.
- Join message is a good idea, however a better solution would be to send a message to the guild owner instead of a potentially unavailable indexed text channel.
- Unrelated comments should be removed before submitting a PR
- Typo: "JOIN_MASSAGE" should be "JOIN_MESSAGE"